testnets/testnets.md

```diff
@@ -53,6 +73,28 @@ wasmcli query wasm list-code
 wasmcli keys add mywallet
 ```
 
+# Joining to be launched testnets
+
+You need to have your address and informations defined in networks genesis file to join for not yet launched testnets.
+Here is the script you can run to handle it automatically. It uses `coral` [network specific executables](https://github.com/CosmWasm/testnets/tree/master/coral#coral-wip):
+
+```sh
+cd $CW_DIR
+git clone git@github.com:CosmWasm/testnets
+cd testnets
+git checkout -b add-gen-acc-<validator-name>
+wasmd add-genesis-account $(wasmcli keys show -a mywallet) 100000000ushell,100000000ustake
```

Comment:
Does this work or need the homedir to be set?

Ahh... I see the work in the next two lines. You can set this up with two steps.

1. store the genesis under `testnets/coral/config/genesis.json` (note config dir)
2. Run this with home set to current dir, so it will edit that file directly: `wasmd add-genesis-account --home $(pwd) $(wasmcli keys show -a mywallet) 100000000ushell,100000000ustake`